config.js

```diff
@@ -20,7 +20,7 @@
  *                                                             *
  ***************************************************************/
 
-export const USE_REMOTE_MODELS = false;
+export const USE_REMOTE_MODELS = true;
```

Comment:
It seem this line is auto changed. By "npm run prod:use-remote-models"?

Reply:
Yes, we'd better not track the modification on the commit

Reply:
Reverted the change in a new PR. PTAL.